control-plane/api-gateway/binding/validation.go

```diff
@@ -17,6 +16,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	gwv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 	gwv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+
+	"github.com/hashicorp/consul-k8s/control-plane/api-gateway/common"
+	"github.com/hashicorp/consul-k8s/control-plane/api/v1alpha1"
+	"github.com/hashicorp/consul-k8s/control-plane/version"
+	"github.com/hashicorp/consul/api"
```

Comment:
#nit doesn't follow the blocking pattern established in other files in this PR

I'm not sure if we have a consul-wide standard. I was wondering this myself the other day 🤔

Reply:
I can't find the actual docs now, but for some reason I added this to my goimports setup when I started :question: https://github.com/jm96441n/dotfiles/blob/master/.config/nvim/lua/plugins/config/formatter.lua#L16